exasol/github.py

```diff
@@ -0,0 +1,78 @@
+"""
+Github-related utility functions - check for latest release of
+project, retrieval of artefacts, etc.
+"""
+import enum
+import requests
+import pathlib
+import logging
+from typing import Tuple, Optional
+
+_logger = logging.getLogger(__name__)
+
+
+class Project(enum.Enum):
+    """
+    Names of github projects to be retrieved. Values have to
+    match github project names.
+    """
+    CLOUD_STORAGE_EXTENSION = "cloud-storage-extension"
+    KAFKA_CONNECTOR_EXTENSION = "kafka-connector-extension"
+
+
+def get_latest_version_and_jar_url(project: Project) -> Tuple[str, str]:
+    """
+    Retrieves the latest version of stable project release
+    and url with jar file from the release.
+
+    :param project: name of the project
+    :return: tuple with version and url to retrieve the artefact.
+    """
+    req = requests.get(f"https://api.github.com/repos/exasol/{project.value}"
+                       f"/releases/latest", timeout=10)
+    if req.status_code != 200:
+        raise RuntimeError("Error sending request to the github, code: %d" %
+                           req.status_code)
```

Comment:
Alternatively, you could write:
```suggestion
    try:
        req.raise_for_status()
    except HTTPError as ex:
        raise RuntimeError(
            f"Error sending request to the github, code: {req.status_code}"
         ) from ex
```

Reply:
or if you would be fine with the HTTPError simply
```python
        req.raise_for_status()
```
would do